packaging/hipify-clang.txt

```diff
@@ -12,9 +12,9 @@ set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc.")
 set(CPACK_PACKAGE_VERSION_MAJOR @LLVM_VERSION_MAJOR@)
 set(CPACK_PACKAGE_VERSION_MINOR @LLVM_VERSION_MINOR@)
 set(CPACK_PACKAGE_VERSION_PATCH @LLVM_VERSION_PATCH@)
-#TODO : Change mail id to standard <package.suppport@amd.com>
-set(CPACK_PACKAGE_CONTACT "Mankov Evgeny <Evgeniy.Mankov@amd.com>")
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "hipify-clang: a clang-based tool to translate CUDA source code into portable HIP C++ automatically")
+set(CPACK_PACKAGE_CONTACT "ROCm Compiler Support <rocm.compiler.support@amd.com>")
```

Comment:
Is `rocm.compiler.support@amd.com` a valid e-mail?

Reply:
It is.  Mark created it. We're using it for all ROCm compiler components.